doc/rados/troubleshooting/log-and-debug.rst

```diff
@@ -200,59 +200,105 @@ to their default level or to a level suitable for normal operations.
 +--------------------+-----------+--------------+
 | ``mon``            |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``monc``           |     0     |      5       |
+| ``monc``           |     0     |      10      |
 +--------------------+-----------+--------------+
-| ``paxos``          |     0     |      5       |
+| ``paxos``          |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``tp``             |     0     |      5       |
 +--------------------+-----------+--------------+
 | ``auth``           |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``finisher``       |     1     |      5       |
+| ``crypto``         |     1     |      5       |
++--------------------+-----------+--------------+
+| ``finisher``       |     1     |      1       |
++--------------------+-----------+--------------+
+| ``reserver``       |     1     |      1       |
 +--------------------+-----------+--------------+
 | ``heartbeatmap``   |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``perfcounter``    |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``rgw``            |     1     |      5       |
 +--------------------+-----------+--------------+
+| ``rgw sync``       |     1     |      5       |
++--------------------+-----------+--------------+
+| ``civetweb``       |     1     |      10      |
++--------------------+-----------+--------------+
 | ``javaclient``     |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``asok``           |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``throttle``       |     1     |      5       |
+| ``throttle``       |     1     |      1       |
++--------------------+-----------+--------------+
+| ``refs``           |     0     |      0       |
++--------------------+-----------+--------------+
+| ``xio``            |     1     |      5       |
++--------------------+-----------+--------------+
+| ``compressor``     |     1     |      5       |
++--------------------+-----------+--------------+
+| ``bluestore``      |     1     |      5       |
++--------------------+-----------+--------------+
+| ``bluefs``         |     1     |      5       |
++--------------------+-----------+--------------+
+| ``bdev``           |     1     |      3       |
++--------------------+-----------+--------------+
+| ``kstore``         |     1     |      5       |
++--------------------+-----------+--------------+
+| ``rocksdb``        |     4     |      5       |
++--------------------+-----------+--------------+
+| ``leveldb``        |     4     |      5       |
++--------------------+-----------+--------------+
+| ``memdb``          |     4     |      5       |
++--------------------+-----------+--------------+
+| ``kinetic``        |     1     |      5       |
```

Comment:
might want to remove the `kinetic` entry here. it's not built anymore.

Reply:
@tchaikov Thanks for your review.
BTW where can i find the default build rule about the loging entry?

Reply:
could you be more specific w.r.t. "build rule about the logging entry"?

Reply:
@tchaikov Ok, I update the table entry info from src/common/subsys.h 
So if we don't need "kinetic", maybe we should remove "SUBSYS(kinetic, 1, 5)" too ?